### PR TITLE
layer.md: fix spelling error

### PR DESCRIPTION
**Pull request by TrumanLing**
Signed-off-by: Ling FaKe <lingfake@huawei.com>

### CODE REVIEW

File: layer.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ The corresponding files that share the link with the > 1 linkcount may be outsid
 
 Hardlinks are stored in a tar archive with type of a `1` char, per the [GNU Basic Tar Format][gnu-tar-standard] and [libarchive tar(5)][libarchive-tar].
 
-While approaches to deriving new or changed hardlinks may vary, a possible approach is:
+While approaches to derive new or changed hardlinks may vary, a possible approach is:
```
Comment (jonboulle) [inline layer.md]: The original was correct

File: layer.md [hunk 2]
```diff
@@ -204,7 +204,7 @@ The resulting tar archive for `rootfs-c9d-v1.s1` has the following entries:
 ./etc/.wh.my-app-config
 ```
 
-To signify that the resource `./etc/my-app-config` MUST be removed when the changeset is applied, the basename of the entry is prefixed with `.wh.`.
+To signify that, the resource `./etc/my-app-config` MUST be removed when the changeset is applied, the basename of the entry is prefixed with `.wh.`.
```
Comment (jonboulle) [inline layer.md]: This changes the meaning. "that" is used as a conjuction here, not a pronoun.

**Comment by jonboulle** [comment]
Sorry, neither of these are errors.

File: layer.md (hunk 1 above)
Comment (TrumanLing) [inline layer.md]: I think it should be ```to derive``` or ```for deriving```.

File: layer.md (hunk 2 above)
Comment (TrumanLing) [inline layer.md]: As for here, I agree with you.

Comment (TrumanLing) [inline layer.md]: As for here, I agree with you.

**Pull request closed without merging**